packages/sdk/src/evm/contracts/prebuilt-implementations/pack.ts

```diff
@@ -287,22 +288,22 @@ export class Pack extends StandardErc1155<PackContract> {
             tokenMetadata.decimals,
           );
           erc20Rewards.push({
-            contractAddress: reward.assetContract,
+            contractAddress: reward.assetContract as Address,
```

Comment:
wait does this mean TS will yell at me for passing a string address now?

Reply:
No, `Address` is just a `string` type. But I initially made it with address as `0x${string}` just to have errors propagate up and try to make the repo compatible with `Address` / `AddressOrEns` types regardless of if the types switch out underneath. But switched it back to just `string` to prevent the user type errors